tasks/broad/JointGenotypingTasks.wdl

```diff
@@ -149,7 +149,7 @@ task GenotypeGVCFs {
     Int disk_size
     # This is needed for gVCFs generated with GATK3 HaplotypeCaller
     Boolean allow_old_rms_mapping_quality_annotation_data = false
-    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.1.8.0"
+    String gatk_docker = "us.gcr.io/broad-dsde-methods/ggvcfs_on_reblocked@sha256:09ffced9cd5457effb54cf83062c628d9caa885addb025d1c7e1589033e83a36"
```

Comment:
Why this docker and not "us.gcr.io/broad-gatk/gatk:4.2.2.0"?

Reply:
I need a 4.2.2.1... hopefully today.